pkg/proc/proc.go

```diff
@@ -466,12 +466,15 @@ func StepOut(dbp Process) error {
 
 // GoroutinesInfo returns an array of G structures representing the information
 // Delve cares about from the internal runtime G structure.
```

Comment:
Update documentation to cover new arguments and return values.

Reply:
Sure, how does something like this look to you?

```
// GoroutinesInfo searches for goroutines starting at index 'start', and
// returns an array of up to 'count' (or all found elements, if 'count' is 0)
// G structures representing the information Delve care about from the internal
// runtime G structure.
// GoroutinesInfo also returns the next index to be used as 'start' argument
// while scanning for all available goroutines, or -1 if there was an error
// or if the index already reached the last possible value.
```

Reply:
LGTM